### PR TITLE
Add readme explanation re usage in docker-compose

### PR DESCRIPTION
**Pull request by unfinishedteleporter**
As per https://github.com/ufoscout/docker-compose-wait/issues/8

### CODE REVIEW

**Comment by jonasbn** [comment]
Oh could I review the proposed addition?

**Comment by ufoscout** [comment]
@unfinishedteleporter 
Thank you for this PR. It looks good to me.

@jonasbn 
sure! your feedback will be appreciated. Let me know.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,11 @@ services:
 When docker-compose is started (or Kubernetes or docker stack or whatever), your application will be started only when all the pairs host:port in the WAIT_HOSTS variable are available.
 The WAIT_HOSTS environment variable is not mandatory, if not declared, the script executes without waiting.
 
+Note that if you wish to use the script directly in the docker-compose.yml file instead of the Dockerfile, you will need to use an approach like the ones disccussed [here](https://stackoverflow.com/questions/30063907/using-docker-compose-how-to-execute-multiple-commands) and [here](https://github.com/docker/compose/issues/2033) because the `command:` configuration option can only execute a single command. For example:
```
Comment (ufoscout) [inline README.md]: typo: disccussed -> discussed

Comment (unfinishedteleporter) [reply README.md]: Oops :)

[the remark this answers is not in the document]
Comment (jonasbn) [reply README.md]: @unfinishedteleporter do my comments make sense?

[the remark this answers is not in the document]
Comment (unfinishedteleporter) [reply README.md]: That makes sense. Reworded.

**Comment by ufoscout** [comment]
@unfinishedteleporter @jonasbn 
thank you, guys!

**Pull request merged**